js/main.js

```diff
@@ -0,0 +1,30 @@
+var $userPhotoUrl = document.querySelector('#photo-url');
+var $placeHolderImg = document.querySelector('#placeholder');
+
+$userPhotoUrl.addEventListener('input', function (event) {
+  $placeHolderImg.setAttribute('src', event.target.value);
+});
+
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var formValueObject = {};
+
+  var titleKey = event.target.elements[0];
```

Comment:
You don't need to create new variables on lines 14-16. You can just assign those values directly to your `formValueObject`. 

For example, instead of:

```
var titleKey = event.target.elements[0];
formValueObject.titleKey = titleKey.value;
```

You should be able to do: 
`formValueObject.titleKey = event.target.elements[0].value`